app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,6 @@
 class WelcomeController < ApplicationController
   def index
+    Rollbar.info('Access index')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.